examples/configurable_chain/server.py

```diff
@@ -38,26 +59,46 @@
 )
 chain = prompt | model | StrOutputParser()
 
-app = FastAPI(
-    title="LangChain Server",
-    version="1.0",
-    description="Spin up a simple api server using Langchain's Runnable interfaces",
-)
+# Add routes requires you to specify which config keys are accepted
+# specifically, you must accept `configurable` as a config key.
+add_routes(app, chain, path="/config_from_rc", config_keys=["configurable"])
```

Comment:
what's the `rc` in `config_from_rc`?

Reply:
runnable_config. Let me change the name here given it's not clear